plutus-core/plutus-core/src/PlutusCore/Builtin/Meaning.hs

```diff
@@ -294,13 +294,29 @@ instance
     -- See Note [One-shotting runtime denotations].
     -- Grow the builtin application within the received action and recurse on the result.
     toMonoF getBoth = BuiltinExpectArgument . oneShot $ \arg ->
-        -- Ironically computing the unlifted value strictly is the best way of doing deferred
```

Comment:
> Ironically computing

This sounds like some kind of technique: "ironic computation".  We should invent something just so we can call it that.

Reply:
Love the idea.